cmd/rm-main.go

```diff
@@ -121,17 +134,34 @@ func checkRmSyntax(ctx *cli.Context) {
 	isForce := ctx.Bool("force")
 	isRecursive := ctx.Bool("recursive")
 	isStdin := ctx.Bool("stdin")
-
+	isDangerous := ctx.Bool("dangerous")
+	isNamespaceRemoval := false
+	for _, url := range ctx.Args() {
+		url = path.Clean(url)
```

Comment:
It is required to perhaps bring change like this instead `filepath.ToSlash(filepath.Clean(url))` here, for windows paths such as `C:\` path.Clean will not work.

Also question for me why is this path.Clean needed? is it for isAliasURLDir() ? A comment would be needed i think here.

Reply:
@y4m4,  on further thinking, the path.Clean and isAliasURLDir() check are superfluous and can be dispensed with altogether.

Reply:
Okay that would be better.